cpp/src/rolling/grouped_rolling.cu

```diff
@@ -127,8 +127,9 @@ std::unique_ptr<column> grouped_rolling_window(table_view const& group_keys,
   //   groups.)
   //   3. [0, 500, 1000] indicates two equal-sized groups: [0,500), and [500,1000).
 
-  assert(group_offsets.size() >= 2 && group_offsets[0] == 0 &&
-         group_offsets[group_offsets.size() - 1] == input.size() &&
+  assert(group_offsets.size() >= 2 && group_offsets.element(0, rmm::cuda_stream_default) == 0 &&
+         group_offsets.element(group_offsets.size() - 1, rmm::cuda_stream_default) ==
+           input.size() &&
```

Comment:
Why not use the input `stream` parameter here?

Reply:
Hmm... I could have sworn there wasn't a `stream` parameter available at this point. Will fix this shortly.